scripts/app.js

```diff
@@ -7,22 +7,14 @@ function ProjectItem (object){
   this.creators = object.creators;
   this.date = object.date;
   this.repoUrl = object.repoUrl;
+  this.location = object.location;
   this.summary = object.summary;
 };
```

Comment:
Let's refactor this so that you don't have to list out every property separately.